tol_lab_share/messages/traction/qc_message.py

```diff
@@ -202,7 +203,13 @@ def send(self, url: str) -> bool:
         """
         headers = {"Content-type": "application/vnd.api+json", "Accept": "application/vnd.api+json"}
 
-        r = post(url, headers=headers, data=dumps(self.payload(), default=str), verify=self._validate_certificates)
+        try:
+            r = post(url, headers=headers, data=dumps(self.payload(), default=str), verify=self._validate_certificates)
+        except Exception as ex:
+            logger.critical(f"Error submitting {self.__class__.__name__} to the Traction API.")
```

Comment:
Guessing critical is used to make sure its above any minimum logger level set?

Reply:
Hmm yeah I chose this because Crawler logs at this level for this problem.  I agree that it doesn't seem like a critical problem though, does it?  I think perhaps critical intends the problem to be published to a Slack channel or something.  So perhaps the log level is just being misused.  Maybe we should discuss next week.

Reply:
I discussed with the rest of the team, and we've decided to go with what we have here and change it if needed later.